cli/readme.py

```diff
@@ -538,7 +542,7 @@ def write_job_using_registry_components_workflow(job):
       run: |
           echo '{GITHUB_CONCURRENCY_GROUP}';
           bash bootstrap.sh
-      working-directory: infra
+      working-directory: infra/bootstrapping
```

Comment:
Was this incorrect before?

Reply:
Not that I'm aware of. I noticed this change in some of the files and will revert it before final review/merge.